packages/react-native/package.json

```diff
@@ -42,7 +42,6 @@
   },
   "dependencies": {
     "@aws-amplify/react-native": "console-preview",
```

Comment:
Not sure where this was added, but why do we need this package as a direct dep?

Reply:
Good catch, also not sure how that got in there but definitely removing it.